conduit/plugins/importers/filereader/filereader_test.go

```diff
@@ -70,7 +79,7 @@ func initializeTestData(t *testing.T, dir string, numRounds int) sdk.Genesis {
 			Certificate: nil,
 		}
 		blockFile := path.Join(dir, fmt.Sprintf(filewriter.FilePattern, i))
-		err = filewriter.EncodeJSONToFile(blockFile, block, true)
+		err = filewriter.EncodeToFile(blockFile, block, defaultEncodingFormat, defaultIsGzip)
```

Comment:
do we need to define `defaultEncodingFormat`, and `defaultIsGzip` are consts in this file? It looks like we could replace these consts with `filewrite.MessagepackFormat` and `true` and remove `TestDefaults`.

Reply:
I'm aiming for an explicit declaration that these are the defaults, in case this changes in the future. However, `TestDefaults()` isn't actually asserting that the defined defaults are accurate. I've fixed this here: fb7ea14e340df39f64b1c7281a19fb36a7f1ad40